core/variant/variant.h

```diff
@@ -693,6 +693,25 @@ const Variant::ObjData &Variant::_get_obj() const {
 	return *reinterpret_cast<const ObjData *>(&_data._mem[0]);
 }
 
-String vformat(const String &p_text, const Variant &p1 = Variant(), const Variant &p2 = Variant(), const Variant &p3 = Variant(), const Variant &p4 = Variant(), const Variant &p5 = Variant());
+template <typename... Vs>
+String vformat(const String &p_text, const Vs &... vs) {
+	Array args;
+	collect_args(args, vs...);
+	bool error = false;
+	String fmt = p_text.sprintf(args, &error);
 
-#endif // VARIANT_H
+	ERR_FAIL_COND_V(error, String());
+
+	return fmt;
+}
+
+void collect_args(const Array &);
```

Comment:
Need to do this or else c++ will complain you have multiple definition (declaration?) of this function.
Haven't programmed in C++ for a while.

Reply:
Declaration is unnecessary if you define method in the same file.

Reply:
if i define it here, its going to complain that there are multiple definitions of this.